CHANGELOG.md

```diff
@@ -0,0 +1,61 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) 
+(modification: no type change headlines) and this project adheres to 
+[Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+
+## [5.1.2] - 2017-05-31
+- Add browserify for ``ES2015`` compatibility
+- Fix hex validation
+
+[5.1.2]: https://github.com/ethereumjs/ethereumjs-util/compare/v5.1.1...v5.1.2
+
+## [5.1.1] - 2017-02-10
+- Use hex utils from ``ethjs-util``
+- Move secp vars into functions
+- Dependency updates
+
+[5.1.1]: https://github.com/ethereumjs/ethereumjs-util/compare/v5.1.0...v5.1.1
+
+## [5.1.0] - 2017-02-04
+- Fix ``toRpcSig()`` function
+- Updated Buffer creation (``Buffer.from``)
+- Dependency updates
+- Fix npm error
+- Use ``keccak`` package instead of ``keccakjs``
+- Helpers for ``eth_sign`` RPC call
+
+[5.1.0]: https://github.com/ethereumjs/ethereumjs-util/compare/v5.0.1...v5.1.0
+
+## [5.0.1] - 2016-11-08
+- Fix ``bufferToHex()``
+
+[5.0.1]: https://github.com/ethereumjs/ethereumjs-util/compare/v5.0.0...v5.0.1
+
+## [5.0.0] - 2016-11-08
+- Added ``isValidSignature()`` (ECDSA signature validation)
+- Change ``v`` param in ``ecrecover()`` from ``Buffer`` to ``int`` (breaking change!)
+- Fix property alias for setting with initial parameters
+- Reject invalid signature lengths for ``fromRpcSig()``
+- Fix ``sha3()`` ``width`` param (byte -> bit)
+- Fix overflow bug in ``bufferToInt()``
+
+[5.0.0]: https://github.com/ethereumjs/ethereumjs-util/compare/v4.5.0...v5.0.0
+
+## [4.5.0] - 2016-17-12
```

Comment:
17th month of the Mayan calendar 😉

Reply:
I didn't want to be bee too restrictive in calendar choice! :-)